config/checkstyle/checkstyle.xml

```diff
@@ -230,12 +230,13 @@
         </module>
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
-            <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
-            <property name="allowMissingReturnTag" value="true"/>
+            <property name="allowMissingParamTags" value="false"/>
+            <property name="allowMissingThrowsTags" value="false"/>
```

Comment:
![image](https://user-images.githubusercontent.com/60384540/97529641-4de82300-19eb-11eb-9d74-731ef4596ca3.png)
Inconsistent arrow used (dotted and solid arrow) and the arrow head in the circled area is over the box which should have only been hit the box only.
Lack of system architecture diagram to show the overall design of the program. (only uml and sequence diagram is available).

For this diagram, it is hard to interpret if it is a uml class diagram as the boxes used are wrong and format of the diagram is not consistent with the format of a  standard uml class diagram.